modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -321,6 +321,11 @@ export interface RecoverTokenTransaction {
   };
 }
 
+export interface TokenEnablementConfig {
+  requiresTokenEnablement: boolean;
```

Comment:
add doc: "flag indicating if tokens need to be enabled before receiving and sending"